packages/types/src/NetworkInterfaces.ts

```diff
@@ -2,12 +2,78 @@ import { IAccount } from './IAccount';
 import { IStorageEngine } from './IStorageEngine';
 import { SecureContextConfig } from './StorageLinkInterfaces';
 
-export enum EnvironmentType {
-    LOCAL = "local",
+export enum Network {  
+	LOCAL = "local",  
 	DEVNET = "devnet",
-    TESTNET = "testnet",
-    MAINNET = "mainnet"
+	BANKSIA = "banksia",  
+	MYRTLE = "myrtle",  
 }
+  
+/**  
+ * @todo Deprecate in favour of `Network`
+ */
+export enum EnvironmentType {  
+	LOCAL = Network.LOCAL,  
+	DEVNET = Network.DEVNET,  
+	BANKSIA = Network.BANKSIA,  
+	MYRTLE = Network.MYRTLE,  
+
+	/**  
+	 * @deprecated use MYRTLE instead  
+	 */
+	MAINNET = Network.MYRTLE,  
+	/**  
+	 * @deprecated use BANKSIA instead  
+	 */  
+	TESTNET = Network.BANKSIA,  
+}
+
+export enum BlockchainAnchor {
+	POLAMOY = "polamoy",
+	POLPOS = "polpos",
+
+	/**  
+	 * @todo Remove once Amoy is deployed
+	 */
+	MUMBAI = "mumbai",
+
+	/**  
+	 * @deprecated use POLPOS instead  
+	 */
+	MAINNET = "polpos",
+
+	/**  
+	 * @deprecated use MUMBAI instead  
+	 */
+	TESTNET = "mumbai",
+	//TESTNET = "polamoy",
+	DEVNET = "mumbai"
+}
+
+export enum BlockchainNetworks {
```

Comment:
How about a blockchain definitions similar to the network one where you gather the Verida ids (polpos, ...), the hex ids, the CAIP-2 chain ids, the default RPC URLS, ...?

Reply:
I like this idea, however this refactor is quite complicated already, so I think this might need to be a second iteration of the refactor once everything is working agian.